addons/hr_holidays/static/src/scss/accrual_plan_level.scss

```diff
@@ -55,63 +50,42 @@ $o-hr-holidays-border-color: map-get($grays, '300');
 
             // Whole record
             .o_hr_holidays_body {
-                margin-left: 8px;
-                padding-top: 20px;
-
                 // Left side 'Level'
                 .o_hr_holidays_timeline {
                     @include o-position-absolute($top: 32px, $left: 6px);
-                    width: 90px;
-                    padding: 3px 0px;
-                    border-radius: 3px;
-                    background-color: $o-gray-200;
-                    box-shadow: 0 1px 2px rgba(0,0,0,.1);
+                    width: var(--o_hr_holidays_timeline-width);
                 }
 
                 // Actual kanban card
                 .o_hr_holidays_card {
-                    position: relative;
-                    margin-left: 22px;
-                    margin-right: 2px;
                     width: 500px;
-                    border-radius: 3px;
-                    background-color: $o-gray-200;
-                    box-shadow: 0 1px 2px rgba(0,0,0,.1);
 
                     // Triangle
                     &:before {
                         content: '';
-                        @include o-position-absolute($top: 12px, $left: -17px);
+                        @include o-position-absolute($top: 12px, $left: -18px);
                         margin-left: 10px;
                         width: 14px;
                         height: 14px;
-                        background-color: $o-gray-200;
-                        border-bottom: 1px solid $o-hr-holidays-border-color;
+                        background: $o-view-background-color;
+                        border-bottom: $border-width solid $border-color;
+                        border-left: $border-width solid $border-color;
                         transform: rotate(45deg);
                     }
 
                     // Circle
                     &:after {
+                        --hrHolidays-Circle-Size: 12px;
                         content: '';
-                        @include o-position-absolute($top: 14px, $left: -36px);
-                        width: 12px;
-                        height: 12px;
-                        border: 2px solid $o-brand-primary;
+                        @include o-position-absolute($top: 14px, $left: calc((var(--KanbanRecord-padding-h) + #{$spacer * 1.5} + var(--hrHolidays-Circle-Size) * .5 + #{$border-width * .5}) * -1));
+                        width: var(--hrHolidays-Circle-Size);
+                        height: var(--hrHolidays-Circle-Size);
+                        border: 2px solid map-get($o-theme-text-colors, 'primary');
```

Comment:
commit 771dc529383f66446aa307baada638a0c9ce192c
Why not $border-width * 2, check for other arbitrary values occurrences as well, I think you could simplify more in this file but if the calculations get  too complex I think it could be worth investigating another way of implementing this layout.

Reply:
Same as above, I'll check for arbitrary values but I suggest to visually improve the view in a second PR.